samples/backups-cancel.js

```diff
@@ -13,57 +13,68 @@
+  instanceId = 'my-instance',
+  databaseId = 'my-database',
+  backupId = 'my-backup',
+  projectId = 'my-project-id'
+) {
   // [START spanner_cancel_backup_create]
-  // Imports the Google Cloud client library and precise date library
-  const {Spanner} = require('@google-cloud/spanner');
-
   /**
    * TODO(developer): Uncomment the following lines before running the sample.
    */
-  // const projectId = 'my-project-id';
   // const instanceId = 'my-instance';
   // const databaseId = 'my-database';
   // const backupId = 'my-backup';
+  // const projectId = 'my-project-id';
+
+  // Imports the Google Cloud Spanner client library
+  const {Spanner} = require('@google-cloud/spanner');
 
-  // Creates a client
+  // Instantiates a client
   const spanner = new Spanner({
     projectId: projectId,
   });
 
-  // Gets a reference to a Cloud Spanner instance and database
-  const instance = spanner.instance(instanceId);
-  const database = instance.database(databaseId);
+  async function cancelBackup() {
+    // Gets a reference to a Cloud Spanner instance and database
+    const instance = spanner.instance(instanceId);
+    const database = instance.database(databaseId);
 
-  const backup = instance.backup(backupId);
+    const backup = instance.backup(backupId);
 
-  // Creates a new backup of the database
-  try {
-    console.log(`Creating backup of database ${database.formattedName_}.`);
-    const databasePath = database.formattedName_;
-    // Expire backup one day in the future
-    const expireTime = Date.now() + 1000 * 60 * 60 * 24;
-    const [, operation] = await backup.create({
-      databasePath: databasePath,
-      expireTime: expireTime,
-    });
+    // Creates a new backup of the database
+    try {
+      console.log(`Creating backup of database ${database.formattedName_}.`);
+      const databasePath = database.formattedName_;
+      // Expire backup one day in the future
+      const expireTime = Date.now() + 1000 * 60 * 60 * 24;
+      const [, operation] = await backup.create({
+        databasePath: databasePath,
+        expireTime: expireTime,
+      });
 
-    // Cancel the backup
-    await operation.cancel();
+      // Cancel the backup
+      await operation.cancel();
 
-    console.log('Backup cancelled.');
-  } catch (err) {
-    console.error('ERROR:', err);
-  } finally {
-    // Delete backup in case it got created before the cancel operation
-    await backup.delete();
+      console.log('Backup cancelled.');
+    } catch (err) {
+      console.error('ERROR:', err);
+    } finally {
+      // Delete backup in case it got created before the cancel operation
+      await backup.delete();
 
-    // Close the database when finished.
-    await database.close();
+      // Close the database when finished.
+      await database.close();
+    }
   }
+  cancelBackup().catch(console.error);
   // [END spanner_cancel_backup_create]
 }
-
-module.exports.cancelBackup = cancelBackup;
+main(...process.argv.slice(2));
```

Comment:
Please add 
```ts
process.on('unhandledRejection', err => {
  console.error(err.message);
  process.exitCode = 1;
});
```
Above the call of the `main` function as per the Sample Policy doc.

> When main is called, it should catch any errors as well as any unhandled rejecion errors,

Here is the similar from [storage](https://github.com/googleapis/nodejs-storage/blob/24a08c15c7f35a6b26b84159d9926b9e502a2cdd/samples/addBucketLabel.js#L58-L61) repo

Reply:
Done!